src/roaring.c

```diff
@@ -2447,8 +2375,12 @@ void roaring_bitmap_lazy_xor_inplace(roaring_bitmap_t *x1,
                 ra_set_container_at_index(&x2->high_low_container, pos2, c2,
                                           type2);
             }
-            ra_insert_new_key_value_at(&x1->high_low_container, pos1, s2, c2,
-                                       type2);
+
+            if (!ra_insert_new_key_value_at(&x1->high_low_container,
+                                            pos1, s2, c2, type2))
+            {
```

Comment:
The current style used throughout is, I believe to have the curly brace on the same line as the "if".